pkgs/development/compilers/openjdk/openjfx/15.nix

```diff
@@ -64,8 +64,10 @@ let
     outputHashMode = "recursive";
     # Downloaded AWT jars differ by platform.
     outputHash = {
-      x86_64-linux = "077zss95iq6iskx7ghz1c57ymydpzj0wm7r1pkznw99l9xwvdmqi";
-      i686-linux = "03gglr2sh77cyg16qw9g45ji33dg7i93s5s30hz3mh420g112qa0";
+      x86_64-linux = "0hmyr5nnjgwyw3fcwqf0crqg9lny27jfirycg3xmkzbcrwqd6qkw";
+      # The build-time dependencies don't currently build for i686, so no
+      # reason to fetch this one correctly either...
+      i686-linux = "0000000000000000000000000000000000000000000000000000";
```

Comment:
Hash:

```suggestion
      i686-linux = "0000000000000000000000000000000000000000000000000000";
```

Reply:
These lines are identical according to the GitHub UI ... what is the change you're suggesting here?

Reply:
Never mind :sweat_smile: . Haven't read the comment.